src/emulator/modules/CMakeLists.txt

```diff
@@ -139,3 +139,4 @@ add_library(modules STATIC
 	SceVoiceQoS/SceVoiceQoS.cpp SceVoiceQoS/SceVoiceQoS.h
 	SceVshBridge/SceVshBridge.cpp SceVshBridge/SceVshBridge.h)
 target_link_libraries(modules PRIVATE module)
+target_link_libraries(modules PRIVATE rtc)
```

Comment:
`modules/CMakeLists.txt` is automatically generated, you probably would want to add it to `module` instead.

However, `module` already depends on `host` and `host` depends publicly on `rtc`, so you could just delete this line.